backends/bmv2/common/helpers.h

```diff
@@ -85,7 +268,14 @@ struct ConversionContext {
 
     // for action profile conversion
     Util::JsonArray*                 action_profiles;
-    SharedActionSelectorCheck*       selector_check;
+
```

Comment:
Changed to a map to avoid making ConversionContext a template class. As the SharedActionSelectorCheck needs to be templated.